test/modules/NumberLogger/src/Program.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NumberLogger
+{
+    using System;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        public static void Main()
+        {
+            string countString = Environment.GetEnvironmentVariable("Count");
+            int parsedCount = int.Parse(countString);
+
+            for (int i = 0; i < parsedCount; i++)
+            {
+                Console.WriteLine(i);
+            }
+
+            Task.Delay(-1).Wait();
```

Comment:
This container won't respond when docker asks it to close, so docker will have to forcibly close it after some number of seconds. I think it would be better to use the helpers in Edge.Util and Edge.ModuleUtil